docs_versioned_docs/version-ros1noetic/robots/robots.mdx

```diff
@@ -38,6 +38,7 @@ The extensibility of ROS has also allowed Clearpath to use it to develop the pro
 - [Boxer](indoor_robots/boxer/user_manual_boxer.mdx)
```

Comment:
You should discuss this change with someone from Marketing. These links were added in the initial website release because Marketing was concerned that some Users would not find the Robot pages in the side navigation bar.

Reply:
My mistake. I misread the red highlighting as this section being removed. After further review, the only change is adding a link to the Husky Observer page.